keylime/registrar_client.py

```diff
@@ -33,6 +33,16 @@ class RegistrarData(TypedDict):
 logger = keylime_logging.init_logging("registrar_client")
 api_version = keylime_api_version.current_version()
 
+MANTATORY_FIELDS = ['aik_tpm', 'regcount', 'ek_tpm', 'ip', 'port']
+
+
+def check_mantatory_fields(results):
+    for field in MANTATORY_FIELDS:
+        if field not in results:
+            logger.critical("Error: did not receive %s from Registrar Server." % field)
```

Comment:
Apparently there is an extra `%` at the end of the string instead of a `,`

Reply:
ouch :/